src/solvers/solver_utils.hpp

```diff
@@ -13,11 +13,21 @@
 #ifndef SOLVERS_SOLVER_UTILS_HPP_
 #define SOLVERS_SOLVER_UTILS_HPP_
 
+#include <memory>
 #include <string>
+#include <utility>
 #include <vector>
 
 #include "kokkos_abstraction.hpp"
 
+#define INTERNALSOLVERVARIABLE(base, varname)                                            \
```

Comment:
Since this macro is publicly visible, should it have the `PARTHENON_` prefix? I know that's pretty unwieldy...

Reply:
no, that is a good point. I changed it.